src/AppBundle/Resources/views/Order/pdf.html.twig

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
```

Comment:
I think the filename should be `invoice.html.twig` - it is actually a HTML tempalte.

Reply:
Yep.